doc/safety.md

```diff
@@ -0,0 +1,61 @@
+
+Do not use blocks and chainstate from unknown sources. Attackers could be providing you a false blockchain, i.e. one that is valid but not the blockchain that all other nodes are following.
+
+### RPC
+
+The JSON-RPC interface is intended for local access. By default it will only accept connections from localhost, but it can be configured to be accessed for a wider netmask; e.g. a trusted LAN network. You will need to explicitly allow an IP address to connect to it by using the `rpcallowip=<ip>` option in the _bitcoin.conf_ file. If you set it to 0.0.0.0, it will be open to all IP addresses, but this is not recommended as it is not secure.
+
+If you have a need to access it over an untrusted network like the internet, tunnel it through technology specifically designed for that, such as stunnel (SSL) or a VPN.
+
+[Executable, network access, authentication and string handling](/doc/json-rpc-interface.md#security)
+
+CVE-2018–20587 is an Incorrect Access Control vulnerability that affects all currently released versions of Bitcoin Core, you may be affected by this issue if you have the RPC service enabled (default, with the headless bitcoind), and other users have access to run their own services on the same computer (either local or remote access). If you are the only user of the computer running your node, you are not affected. If you use the GUI but have not enabled the RPC service, you are not affected.
+
+You can read more about the vulnerability here: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-20587
+
+[#12763](https://github.com/bitcoin/bitcoin/pull/12763) added the RPC whitelisting feature that helps enforce application policies for services being built on top of Bitcoin Core. It suggested to not connect your Bitcoin node to arbitrary services, reduce risk and prevent unintended access.
+
+### Wallet
+
+[Encrypting the Wallet](/doc/managing-wallets.md#encrypting-the-wallet)
+
+[Backing Up the Wallet](/doc/managing-wallets.md#backing-up-the-wallet)
+
+Do not buy _wallet.dat_ files from anywhere. There are several websites that sell such files to newbies. Scammers modify _wallet.dat_ file to include a bogus ckey record.
+
+Understanding the difference between legacy and descriptor wallets:
+
+Be careful with RPC commands like `dumpwallet`, `dumpprivkey`, and `listdescriptors` with the "private" argument set to true. These commands reveal private keys, which should never be shared with third parties.
+
+Be careful with commands like `importdescriptor` if suggested by third parties, which can control future addresses created for incoming payments
+
+
+### Other
+
+Be careful about accepting configuration changes and commands from third parties. Here is a non-exhaustive list of things that may go wrong:
+
+_*notify_ config and commandline options accept shell commands to be based on different events in Bitcoin Core. Ensure that the commands are safe either by manual inspection or restrictions/alerts in the system used.
```

Comment:
```suggestion
_*notify_ config and command line options accept shell commands to be based on different events in Bitcoin Core. Ensure that the commands are safe either by manual inspection or restrictions/alerts in the system used.
```

Reply:
Done in `088d99b5e5c9bfd907861ce68692e638ee56448d`